.github/workflows/query-planner-ci.yaml

```
@@ -188,10 +203,25 @@ jobs:
     outputs:
       pullrequest_url: ${{ steps.get_url.outputs.pr_url }}
     steps:
+      - name: Authenticate
+        id: app-token
+        uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)